spec/rails_helper.rb

```diff
@@ -0,0 +1,54 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../../../../config/environment', __FILE__)
+
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+require 'spec_helper'
+require 'rspec/rails'
+
+ActiveRecord::Migration.maintain_test_schema!
+
+RSpec.configure do |config|
+
+  config.fixture_path = "#{::Rails.root}/test/fixtures"
+  config.include FactoryGirl::Syntax::Methods
+
+  config.before :suite, type: :feature do
+    if ENV['DRIVER'] == 'selenium'
+      require 'selenium-webdriver'
+    else
+      require 'capybara/poltergeist'
+      Capybara.register_driver :poltergeist do |app|
+        Capybara::Poltergeist::Driver.new(app, js_errors: true,
+                                               inspector: true,
+                                               phantomjs_options: ['--ignore-ssl-errors=yes'],
+                                               timeout: 120
```

Comment:
__[rubocop]__
* Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.

<!-- Comment by [SideCI](https://sideci.com) -->